image/helpers.go

```diff
@@ -53,11 +46,77 @@ type Store interface {
 	Assertion(assertType *asserts.AssertionType, primaryKey []string, user *auth.UserState) (asserts.Assertion, error)
 }
 
+// ToolingStore wraps access to the store for tools.
+type ToolingStore struct {
```

Comment:
I think we want at some point to extract the full helpers.go or most of it to its own package